client/src/services/articlesSorter.spec.js

```diff
@@ -1,27 +1,18 @@
-import articlesSorter from '@/services/articlesSorter';
+import articlesSorter from './articlesSorter'
 
 describe('Unit | services | articlesSorter', () => {
-  beforeEach(() => {
-  });
-
-  afterEach(() => {
-  });
-
   describe('#sortByDropboxId', () => {
     it('should return sortByDropboxId articles', () => {
-      // given
       const article = (dropboxId = 59) => ({
         dropboxId,
         frTitle: 'Le titre',
         enTitle: 'The title',
-      });
-      const articles = [article('2'), article('11'), article(3)];
+      })
+      const articles = [article('2'), article('11'), article(3)]
```

Comment:
id should be string

Reply:
non en fait, mais le test doit expliciter le fait que on ait des string et number